components/camel-irc/src/main/java/org/apache/camel/component/irc/IrcConfiguration.java

```diff
@@ -56,6 +56,10 @@
     @UriParam
     private String nickname;
     @UriParam
```

Comment:
I think it makes sense to add the label for these params without label so it can reflect on the docs